File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void){
    int height, blocks, spaces;
    
    do{
        printf("Height: ");
        height = get_int();
    }while(height > 23 || height < 0);
    
    for(int lines = 1; lines <= height; lines++){
        spaces = height - lines;
        blocks = lines;
        
        for(int i = 0; i < spaces; i++){
            printf(" ");
        }
        
        for(int i = 0; i < blocks; i++){
            printf("#");
        }
        
        printf("  ");
        
        for(int i = 0; i < blocks; i++){
            printf("#");
        }
        
        printf("\n");
    }
}

File: pset6/mario.py
# import modules
import cs50


# main
def main():
    # get height
    while True:
        print("Height: ", end="")
        h = cs50.get_int()
        if h <= 23 and h >= 0:
            break

    # print pyramid
    for b in range(1, h + 1):
        blocks = ("#" * b)
        spaces = (" " * (h - b))
        print("{}{}  {}{}".format(spaces, blocks, blocks, spaces))

    # done
    exit(0)


# redirect to main
if __name__ == "__main__":
    main()
